lightning/src/ln/blinded_payment_tests.rs

```diff
@@ -2460,3 +2460,165 @@ fn test_trampoline_forward_rejection() {
 		expect_payment_failed_conditions(&nodes[0], payment_hash, false, payment_failed_conditions);
 	}
 }
+
+#[test]
+fn test_unblinded_trampoline_forward() {
```

Comment:
Interesting that you do an unblinded forward. I know it has to be supported, but it isn't what an LDK sender would do right?